src/app/common/WarningIcon.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <KIcon
+    icon="warning"
+    color="#0b172d"
+    secondary-color="#ffd68c"
```

Comment:
Seeing as this is all about using the new tokens, I wondered why we had some hex codes in here? Are these colors not in the design tokens module?

Reply:
That’s the case, yes. The tokens aren’t meant to provide a general color palette so very specific things like colors for specific icons are not part of it. Since it was simple to isolate that case like I did, it seems fine to me to use hard-coded colors.